data_table.py

```diff
@@ -0,0 +1,31 @@
+import pandas as pd
+import numpy as np
+import dash
+from dash import dash_table, html
+
+df= pd.read_csv('csv/customers-1000.csv')
```

Comment:
It helps to prepare in case something fails, for example if the file doesn't exists or it don't contains the correct data. 

For these cases, a try and catch is handy to inform the users that something goes wrong.